content/en/docs/reference/scheduling/config.md

```diff
@@ -1,9 +1,17 @@
 ---
-title: Scheduling Profiles
+title: Scheduling Configuration
 content_template: templates/concept
 weight: 20
 ---
 
+## What is new
+
+- `BindTimeoutSeconds` field in scheduler `KubeSchedulerConfiguration` is removed in v1beta1 version, please configure it via plugin args for `VolumeBinding`.
+- `v1beta1` Scheduler `Extender` encoding is now strict, its `HTTPTimeout` is `metav1.Duration` and `enableHttps` is renamed to `enableHTTPS`.
+- `RequestedToCapacityRatioArgs` decoding is now strict
```

Comment:
I recommend against mentioning “new” in headings, at least outside of release notes. Wording like this doesn't age well.

Reply:
Any guidance on how to format release notes instead?

Reply:
Release notes? That's more a SIG Release specialty I think. I don't myself know how those are implemented.

Reply:
By release notes, I mean that this information would have a better home on https://relnotes.k8s.io/ and in https://kubernetes.io/docs/setup/release/notes/

Reply:
We have a series of PRs that in theory require a release note each. It was suggested here https://github.com/kubernetes/kubernetes/pull/91580#issuecomment-637774454 that we accumulate the changes in a website page and point to them. This with the intention of having all the migration actions in a single place.

What do you think about naming the section: `Migrating from v1alpha2 to v1beta1`?

Reply:
Yes, this would be better phrased "Changes between v1alpha2 and v1beta1"

Reply:
Remember to reference field names as they appear in the config file (with the json tag names, not the go struct names)

Reply:
OK, so:
- the release notes in [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) summarize the changes to scheduling for v1.19
- this PR adds more detailed information that those release notes can link to

Have I got that right?

If you want to add extra “What's new in scheduling” details that feel too big to go in a release note, would a blog post work?

Reply:
a reference doc for migrating from v1alpha2 to v1beta1 doesn't seem like it belongs in a blog post... it's an enduring reference that we want in a fixed location as long as this config version is around

Reply:
Hi @sftim I'm on the release notes team for 1.19. I agree with @liggitt . While informative, release notes should be a rather short, to the point item in the list of the user facing changes. The whole reference doc cannot be included in the release notes website but for completeness, I'd suggest adding a link to it in the release note (see [here](https://kubernetes.io/docs/setup/release/notes/#kube-apiserver) for an example).

Reply:
> Remember to reference field names as they appear in the config file 

Will do. I want to have agreement on the section first.
`Changes between v1alpha2 and v1beta1` sgtm.

cc @saschagrunert @puerco from sig-release

Reply:
> Changes between v1alpha2 and v1beta1

That heading works for me.